### DISCUSSION TITLE
Welcome to modmesh Discussions!

### DISCUSSION THREAD

**Discussion by yungyuc** [solvcon/modmesh#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tai271828** [comment]
Hey everyone, I am Tai. I am interested in the "translation" between source code and physics/math equations. I can do any kind of software development from low-level kernel development to high-level ones like desktop, web, and/or cloud-native application development.
